Packs/HostIo/Integrations/HostIo/hostio.py

```diff
@@ -0,0 +1,213 @@
+from typing import List, Any
+import urllib3
+from CommonServerPython import *
+from math import ceil
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def get_domain_data(self, domain: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'full/{domain}',
+            params={}
+        )
+
+    def get_search_data(self, field: str, value: str, limit: int, page: int) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'domains/{field}/{value}',
+            params={'limit': limit,
+                    'page': page}
+        )
+
+    def test_module(self):
+        return self._http_request(
+            method='GET',
+            url_suffix='domains/ip/8.8.8.8',
+            params={}
+        )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module_command(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param client: the base client
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    try:
+        client.test_module()
+    except DemistoException as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return 'ok'
+
+
+def domain_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """domain command: Returns domain reputation for a list of domains
+
+    :type client: ``Client``
+    :param client: Hostio client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['domain']`` list of domains or a single domain
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains Domains
+
+    :rtype: ``CommandResults``
+    """
+    domains = argToList(args.get('domain'))
+
+    command_results: List[CommandResults] = []
+    for domain in domains:
+        domain_data = client.get_domain_data(domain)
+
+        if domain_data.get('web', {}).get('date'):
+            domain_date_dt = dateparser.parse(domain_data['web']['date'])
+            if domain_date_dt:
+                domain_data['updated_date'] = domain_date_dt.strftime(DATE_FORMAT)
+
+        score = Common.DBotScore.NONE
+        dbot_score = Common.DBotScore(
+            indicator=domain,
+            integration_name='HostIo',
+            indicator_type=DBotScoreType.DOMAIN,
+            score=score
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
here and anywhere similar
```suggestion
            score=score,
```